packages/postgraphile-core/__tests__/fixtures/mutations/mutation-return-types.graphql

```diff
@@ -110,6 +110,6 @@ mutation {
     }
   }
   mutationReturnsTableOneCol(input: { i: 20 }) {
-    integers
+    col1S
```

Comment:
I hate this camel case pluralize bug. Oh well, can't fix it now (except by using a plugin).